julia/sanitycheck.jl

```diff
@@ -0,0 +1,105 @@
+using HDF5
+
+export isSane
+
+function isSane(filename::AbstractString; Debug=true)
```

Comment:
@hofmannmartin: in julia this function is usually called `isvalid` although this would require some kind of MDF type which we don't have here. There is also `is_valid_char` in base, so probably a good name would be `isvalid_mdf`.